WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -206,6 +206,13 @@ private extension ProductFormViewController {
         present(alert, animated: true, completion: nil)
     }
 
+    func viewProductInStore() {
```

Comment:
I wonder if we should consistently use `display` since the methods above use it. I think I reviewed the PR for the `shareProduct()` below too so, yeah, I'm inconsistent. 😄 

```suggestion
    func displayWebViewForProductInStore() {
```

I'm not sure if this is a better name though.

Reply:
Yes, I also prefer to use the `display*` prefix 👍  I updated this and also the `shareProduct:` method.